src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/confirm/containers/Confirm.test.tsx

```diff
@@ -9,6 +9,7 @@ import { renderWithProviders } from '../../../../testUtils';
 import Confirm, {
   returnConfirmSummaryObject,
 } from 'src/features/confirm/containers/Confirm';
+import { IParty } from 'altinn-shared/types';
```

Comment:
Would be good to start using `import type` when importing TS types/interfaces (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export). This gives the compiler explicit information that this import should not be part of the final compiled code.



```suggestion
import type { IParty } from 'altinn-shared/types';
```

Reply:
Ah, neat. Thanks!